soc/bouffalolab/bl6/soc.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021-2024 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Board configuration macros
+ *
+ * This header file is used to specify and describe board-level aspects
+ */
+
+#ifndef _SOC__H_
```

Comment:
please remove soc.h entirely, it is not needed for risc-v

Reply:
This is used on the version that uses SDK 1.4.2 to propagate common constants.